thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/DetectionResourceHttpUtils.java

```diff
@@ -0,0 +1,67 @@
+package com.linkedin.thirdeye.anomaly.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.HttpHost;
+import org.apache.http.HttpRequest;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.util.EntityUtils;
+
+/**
+ * Utility classes for calling detector endpoints to execute/schedule jobs
+ */
+public class DetectionResourceHttpUtils {
+
+  private HttpHost detectionResourceHttpHost;
+
+  private static String DETECTION_JOB_ENDPOINT = "/api/detection-job/";
+  private static String ADHOC = "/ad-hoc";
+
+  public DetectionResourceHttpUtils(String detectionHost, int detectionPort) {
+    this.detectionResourceHttpHost = new HttpHost(detectionHost, detectionPort);
+  }
+
+  public String enableAnomalyFunction(String id) throws ClientProtocolException, IOException {
+    HttpPost req = new HttpPost(DETECTION_JOB_ENDPOINT + id);
+    return callDetectionJobEndpoint(req);
+  }
+
+  public String disableAnomalyFunction(String id) throws ClientProtocolException, IOException {
+    HttpDelete req = new HttpDelete(DETECTION_JOB_ENDPOINT + id);
+    return callDetectionJobEndpoint(req);
+  }
+
+  public String runAdhocAnomalyFunction(String id, String start, String end)
```

Comment:
same as before. Convert date in Resource layer itself. (Data conversion should be done at Resource layer itself)